### PR TITLE
Make PermitTunnel configurable

### PR DESCRIPTION
**Pull request by runningman84**
Sometimes you want to enable tunneling, the cookbook should at least allow such kind of configuration. All other "unsecure" options are also allowed to be changed...

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -85,6 +85,8 @@
   server['client_alive_interval']    = 300     # 5min
   server['client_alive_count']       = 3       # ~> 3 x interval
   server['allow_root_with_key']      = false
+   
+  server['permit_tunnel']            = false
```
[the remark this answers is not in the document]
Comment (runningman84) [reply attributes/default.rb]: good catch, just look at my latest update

File: templates/default/opensshd.conf.erb [hunk 2]
```diff
@@ -157,7 +157,7 @@ ClientAliveInterval <%= @node['ssh-hardening']['ssh']['server']['client_alive_in
 ClientAliveCountMax <%= @node['ssh-hardening']['ssh']['server']['client_alive_count'] %>
 
 # Disable tunneling
-PermitTunnel no
+PermitTunnel <%= ((@node['ssh-hardening']['ssh']['server']['permit_tunnel']) ? 'yes' : 'no' ) %>
```
Comment (artem-sidorenko) [inline templates/default/opensshd.conf.erb]: from the man page:
```
     PermitTunnel
             Specifies whether tun(4) device forwarding is allowed.  The argu-
             ment must be yes, point-to-point (layer 3), ethernet (layer 2),
             or no.  Specifying yes permits both point-to-point and ethernet.
             The default is no.
```

So, we have more values then yes and no. How do we want to handle that? Mapping true/false to yes/no and everything else would be passed as-as, @chris-rock @atomic111 @runningman84 your views?

**Comment by bobchaos** [comment]
I see we have no news from the author of this PR but I have a fairly immediate need to see this merged and published. I'm not all that familiar with opensource gitflows, what's the correct process here? Do I open my own fork/PR, apply the requested fixes + cookstyle (i see it failing in travis) or should I clone the fork by runningman and make the fixes there? Let me know how you prefer it and I can get a PR up or updated within the next workday

**Comment by artem-sidorenko** [comment]
@bobchaos please take the patches of @runningman84, set your fixes on top and create another PR.

Please let agree on the open question first, that is the right approach to handle it from your perspective as a user of this feature?

**Comment by bobchaos** [comment]
I think there's no reason we can't support both true/false and yes/no, but the rest of your cookbook takes the boolean approach so we should probably stick to that for the sake of consistency. I'll submit something shortly in a new PR

**Comment by artem-sidorenko** [comment]
closing in favour of #211

**Pull request closed without merging**